apps/test/unit/applab/designModeTest.js

```diff
@@ -165,4 +170,343 @@ describe('setProperty and read Property', () => {
       );
     });
   });
+
```

Comment:
Awesome amount of tests! Since this file is now mostly applab-themes tests, mind moving all these applab-themes tests to a new file?